Dockerfile.chiseled

```diff
@@ -29,6 +29,7 @@ RUN dotnet publish -a $TARGETARCH -c Release -o /app --no-restore --self-contain
 
 # Final stage/image
 FROM mcr.microsoft.com/dotnet/runtime:8.0-jammy-chiseled AS runtime
+
```

Comment:
why is chiseled not installing the required libraries?

Reply:
Following our convo, not needed in chiseled